build.sbt

```diff
@@ -200,6 +224,7 @@ val `example-basic-server` =
 
 val endpoints =
   project.in(file("."))
+    .enablePlugins(CrossPerProjectPlugin)
```

Comment:
It works, but without this line you need to use specific plugin command instead of "+" to perform cross-action(I'm not sure what is actual command, but docs gives `> ;so clean; such test; very publishLocal` as an example). With this line you can just execute `+ publishLocal` and subprojects `crossScalaVersion`s will be taken into consideration.